app-web/src/constants/ui.js

```diff
@@ -31,36 +69,16 @@ export const CARD_CONFIG = {
   avatarIconHeight: 45,
   cardImageWidth: 150,
   cardImageHeight: 150,
-  actionsRibbon: [
-    {
-      name: 'Repository',
-      flag: 'features.card.repository',
-      icon: faGithub,
-      ariaLabel: ARIA_LABEL_REPO,
-      getHref: getGithubRepoRoute,
-    },
-    {
-      name: 'Issues',
-      flag: 'features.card.issues',
-      icon: faComment,
-      ariaLabel: ARIA_LABEL_TO_GITHUB_ISSUES,
-      getHref: getGithubIssuesRoute,
-    },
-    {
-      name: 'Fork',
-      flag: 'features.card.fork',
-      icon: faCodeBranch,
-      ariaLabel: ARIA_LABEL_FORK_GITHUB_REPO,
-      getHref: getGithubForkRoute,
-    },
-    {
-      name: 'Watch',
-      flag: 'features.card.watch',
-      icon: faEye,
-      ariaLabel: ARIA_LABEL_WATCH_GITHUB_REPO,
-      getHref: getGithubWatchRoute,
-    },
-  ],
+  actionsRibbon: {
+    [RESOURCE_TYPES.RESPOSITORIES]: [
```

Comment:
why do you need the []?

Reply:
Oh i'm setting the key to be exactly the same as the enum for resource types
that way if resource types are ever changed this config doesn't need to be changed